### PR TITLE
Fix #176392

### LINKED ISSUE #176392: jovemnerd.com.br
### Issue URL (Annoyance)
 [https://jovemnerd.com.br/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fjovemnerd.com.br%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.35
System version: | Windows 10
Browser: | Chrome
Stealth mode options: | Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Block trackers
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by SpotlightForBugs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

## What issue is being fixed?
https://github.com/AdguardTeam/AdguardFilters/issues/176392

1. Your comment

Removed the cookie banner and resolved the issue mentioned in https://github.com/AdguardTeam/AdguardFilters/issues/176392#issuecomment-2040505140

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

image
</details>
<details>

<summary>Screenshot 2:</summary>

image

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 1]
```diff
@@ -10889,6 +10889,7 @@ stokke.com##.luana-module-cookie-policy
 bitburger.de,otto.de##.m-cookie-layer
 m.krombacher.de##.mainContent > .textSection + .formSection + .textSection
 mini.de##.md-cookie
+jovemnerd.com.br##div:matches-css(z-index: 999):remove()
```
Comment (BlazDT) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:10892]: Too generic.

`jovemnerd.com.br##footer + div[class=""][style*="background-color"][style*="position: fixed"]`

File: BaseFilter/sections/specific.txt [hunk 2]
```diff
@@ -383,6 +383,7 @@ hdzog.com##.content-block > div[class]:not([class*="pag"]):has(> div[class]:only
 zerohedge.com##.main-content .banner
 zerohedge.com##.main-content .native
 zerohedge.com##.main-content #content-pack:empty
+jovemnerd.com.br##.my-10:has(> div:only-child > div[id*="custom-iab"])
```
Comment (BlazDT) [inline BaseFilter/sections/specific.txt:386]: Move to Spanish filter.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:10892]: Also note `#$?#div:matches-css(z-index: 999) { remove: true; }` is our canonical syntax for the expression. However, avoid using `:matches-css()` or pseudo-class `remove` unless absolutely necessary.

File: BaseFilter/sections/specific.txt (hunk 2 above)
Comment (SpotlightForBugs) [reply BaseFilter/sections/specific.txt]: Done

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt (hunk 1 above)
Comment (SpotlightForBugs) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt]: Done

**Review by SpotlightForBugs — COMMENTED** [review]
Done

**Review by Yuki2718 — APPROVED** [review]

**Review by BlazDT — APPROVED** [review]

**Pull request merged**